airbyte-integrations/connectors/source-opsgenie/source_opsgenie/schemas/alerts.json

```diff
@@ -0,0 +1,166 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["null", "string"]
```

Comment:
Old comment from #16056 :
This is an `number` based on the API schema, likely the same for all other `id` cols in all the other streams

Reply:
As with other APIs, the id is a UUID:

```
{
    "data": [
        {
            "seen": false,
            "id": "cdab5d2b-4b5e-49bb-94b7-c7fadcd98b60-1663298614917",
            "tinyId": "53254",
            "alias": "3eb3b201-dd9e-4e12-bc7f-ea7b0ef513a4_5dcd973f-89a2-4aac-a371-997bfa3ce4ce",
            "message": "Testing Incident Reporting",
            "status": "open",
            "acknowledged": false,
            "isSeen": false,
            "tags": [],
            "snoozed": false,
            "count": 1,
            "lastOccurredAt": "2022-09-16T03:23:34.917Z",
            "createdAt": "2022-09-16T03:23:34.917Z",
            "updatedAt": "2022-09-16T03:23:35.019Z",
            "source": "",
            "owner": "Adam Schmidt",
            "priority": "P5",
            "teams": [],
            "responders": [
                {
                    "type": "user",
                    "id": "a9d98683-7ddf-40da-a68b-2eadb54f16e8"
                }
            ],
            "ownerTeamId": ""
        },
        ...
    ],
    "paging": {
        "next": "https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=20&order=desc",
        "first": "https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=0&order=desc",
        "last": "https://api.opsgenie.com/v2/alerts?limit=20&sort=createdAt&offset=20000&order=desc"
    },
    "took": 1.487,
    "requestId": "28a5786e-2afd-4a56-a2c2-3f36284a171c"
}
```